src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/LogsPage/index.html.twig

```diff
@@ -52,11 +52,18 @@
           <div class="card-block row">
             <div class="card-text">
               <div class="form-group row">
-                {% set helptip = 'Enter "5" if you do not want to receive any emails.'|trans({}, 'Admin.Advparameters.Help') ~ ' ' ~ 'Emails will be sent to the shop owner.'|trans({}, 'Admin.Advparameters.Help') %}
-                {{ ps.label_with_help(('Minimum severity level'|trans), helptip, 'col-sm-2') }}
+                {% set helptip_severity = 'Leave empty if you do not want to receive any emails.'|trans({}, 'Admin.Advparameters.Help') ~ ' ' ~ 'Put in the following fields the recipients of these emails.'|trans({}, 'Admin.Advparameters.Help') %}
+                {{ ps.label_with_help(('Minimum severity level'|trans), helptip_severity, 'form-control-label') }}
                 <div class="col-sm">
-                  {{ form_errors(logsByEmailForm) }}
-                  {{ form_widget(logsByEmailForm) }}
+                  {{ form_errors(logsByEmailForm.logs_by_email) }}
+                  {{ form_widget(logsByEmailForm.logs_by_email) }}
+                </div>
+              </div>
+              <div class="form-group row">
+                {% set helptip_receivers = 'The log alerts will be sent to these emails. Please use a comma to separate them (e.g. pub@prestashop.com, anonymous@psgdpr.com).'|trans({}, 'Admin.Advparameters.Help') %}
+                {{ ps.label_with_help(('Send emails to'|trans), helptip_receivers, 'form-control-label') }}
```

Comment:
Domain here is _Admin.Advparameters.Feature_, right?

Reply:
Isn't it a help text @LouiseBonnard ?

Reply:
Nope, it's a feature, you can see it in the screenshot provided. It is the name of the field, 'Send emails to'. :-)

Reply:
Other texts are help texts/tooltips as they will be displayed by the '?' picto.